samcli/commands/init/__init__.py

```diff
@@ -277,13 +277,14 @@ def do_cli(
         if package_type == IMAGE and image_bool:
             base_image, runtime = _get_runtime_from_image(base_image)
```

Comment:
are we now adding a requirement on both base image and app template always needing to be specified. Am I summarizing that correctly.